tests/testapp/models.py

```diff
@@ -282,3 +282,11 @@ class Case(ExtendedModel):
 class Listing(models.Model):
     items = ArrayField(models.TextField(), size=4)
     content_type = models.ForeignKey(ContentType, on_delete=models.PROTECT, blank=True, null=True)
+
+
+class A(models.Model):
+    value = models.CharField(max_length=256, null=True, blank=True)
+
+
+class B(models.Model):
```

Comment:
Can we rename this models in a way that they are descriptive? Perhaps `NullableFk`, and `NullableChar`. This makes it easier to read the code in tests, and states the purpose a bit more clearly.